docs/_includes/projects/docs/lifecycle/first/my-element.js

```diff
@@ -0,0 +1,14 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+class MyElement extends LitElement {  
+  render(){
+    return html`
+      <textarea>Focus me on first update</textarea>
+    `;
+  }
+  firstUpdated(changedProperties){
+    console.log(changedProperties);
```

Comment:
I'm not 100% on this but I believe you're supposed to call `super.firstUpdated(changedProperties)` when you use `firstUpdated`

Reply:
I didn't see anything to suggest this in the code or readme.

Reply:
You're right, my bad